### ISSUE TITLE
save_checkpoints_steps doesnt work.

### ISSUE THREAD

**Issue by dome272** [xuyige/BERT4doc-Classification#10]
The parser option for save_checkpoints_steps doesnt do anything for me.

Im running:

`python3 run_classifier_single_layer.py  --task_name imdb  --do_train  --do_eval  --do_lower_case  --data_dir ./stock  --vocab_file ./uncased_L-12_H-768_A-12/vocab.txt  --bert_config_file ./uncased_L-12_H-768_A-12/bert_config.json  --init_checkpoint ./uncased_L-12_H-768_A-12/pytorch_model.bin  --max_seq_length 512  --train_batch_size 16  --learning_rate 2e-5  --num_train_epochs 3.0  --output_dir ./stock_output  --seed 42  --layers 11 10  --trunc_medium -1 --save_checkpoints_steps 1000`

Any idea to solve this?

**Comment by xuyige** [comment]
Thanks for your issue.
In run_classifier_single_layer.py (or other fine-tuning scripts), we did not save checkpoints
We forgot to remove these unused arguments.
If you want to save checkpoints, you can add your codes during the training phase (e.g., at about Line 930)
